gems/aws-crt-auth/lib/aws-crt-auth/credentials.rb

```diff
@@ -0,0 +1,103 @@
+# frozen_string_literal: true
+
+module Aws
+  module Crt
+    module Auth
+      # Utility class for Credentials.
+      # @api private
+      class Credentials
```

Comment:
trivial: why is this considered private?

Reply:
We consider @api private to just mean - we reserve the right to change the interface to this.  I'd argue that customers shouldn't directly create/depend on this implementation until we have full bindings for credentials/credential providers/credentials resolution chain.  Thoughts?

Reply:
Removed the `@api private` - Agreed this should be public eventually and everything is subject to change during develop preview/pre-release, so I think this is reasonable.